docs/modules/ROOT/pages/documentation/ecosystem/arquillian-containers/payara-server-managed.adoc

```diff
@@ -125,7 +149,72 @@ The easiest way to provide these system properties to the extension runtime is b
     <container qualifier="payara" default="true">
         <configuration>
             <property name="adminPort">4848</property>
+            <property name="adminPassword">${my.admin.password}</property>
         </configuration>
     </container>
 </arquillian>
+----
+
+If you want to configure more containers, you can switch between them by setting the `arquillian.launch` system property to the container's qualifier. This is how you can do it with the maven `surefire` plugin (`my.admin.password` system property is used to set the `adminPassword` property in `arquillian.xml`):
+
+[source,XML]
+.Example Surefire plugin configuration in a `pom.xml` file
+----
+<plugin>
+    <groupId>org.apache.maven.plugins</groupId>
+    <artifactId>maven-surefire-plugin</artifactId>
+    <configuration>
+        <systemPropertyVariables>
+            <arquillian.launch>payara</arquillian.launch>
+            <payara.home>/path/to/payara</payara.home>
+            <my.admin.password>mypassword</my.admin.password>
+        </systemPropertyVariables>
+    </configuration>
+</plugin>
+----
+
+[[automatic-download-example]]
+### Downloading Payara Server automatically from Maven
+
+You can automate downloading and installing Payara Server with maven so that the Arquillian plugin can be automatically configured and used without installing Payara Server manually.
```

Comment:
Bit of a hard read, better to state: "You can configure your Maven project to automatically download and install the Payara Server needed by the Arquillian plugin"

Reply:
```suggestion
You can configure your Maven project to automatically download and install the Payara Server needed by the Arquillian plugin
```